hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/ClientTrustManager.java

```diff
@@ -145,7 +145,7 @@ private void initialize(List<X509Certificate> caCerts)
   private List<X509Certificate> loadCerts(CACertificateProvider caCertsProvider)
       throws CertificateException {
     try {
-      LOG.info("Loading certificates for client.");
+      LOG.debug("Loading certificates for client.");
```

Comment:
Looks like integration test is looking for this log message:

```
Error:  Tests run: 6, Failures: 3, Errors: 0, Skipped: 0, Time elapsed: 11.809 s <<< FAILURE! - in org.apache.hadoop.ozone.container.ozoneimpl.TestOzoneContainerWithTLS
Error:  org.apache.hadoop.ozone.container.ozoneimpl.TestOzoneContainerWithTLS.testLongLivingClientWithCertRenews  Time elapsed: 0.857 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Client loaded certificates. ==> expected: <true> but was: <false>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1156)
	at org.apache.hadoop.ozone.container.ozoneimpl.TestOzoneContainerWithTLS.assertClientTrustManagerLoading(TestOzoneContainerWithTLS.java:288)
	at org.apache.hadoop.ozone.container.ozoneimpl.TestOzoneContainerWithTLS.testLongLivingClientWithCertRenews(TestOzoneContainerWithTLS.java:229)
```

Reply:
@sarvekshayr `TestOzoneContainerWithTLS` needs to set log level for `ClientTrustManager` to `DEBUG`